homeassistant/components/unifi/device_tracker.py

```diff
@@ -173,10 +173,23 @@ async def async_will_remove_from_hass(self) -> None:
         await super().async_will_remove_from_hass()
 
     @callback
-    def async_update_callback(self) -> None:
+    def async_signal_reachable_callback(self) -> None:
+        """Call when controller connection state change."""
+        self.async_update_callback(controller_state_change=True)
+
+    # pylint: disable=arguments-differ
```

Comment:
This is a sign that we should refactor.

Reply:
Yes, I have a few notes of things I should focus on improving thats been introduced during this release phase :)